ember.md

```diff
@@ -98,18 +98,20 @@ CSS is permitted (and encouraged) in apps and addons under certain circumstances
 - Display
  - e.g., `display:flex`, `flex-direction: column`
 - Animations and Transitions
+- Certain Font Styles
+ - `font-weight` only
 
 
 **Use discretion when adding colors to apps/addons**
 * Neutral/non-branding colors that aren't likely to change across websites are OKAY to add to that app's/addon's app styles
-* Font family and line height are NOT okay to update in the app/addon styles
```

Comment:
Only "and line-height" needed to be removed. It is still NOT okay to add font-family. Sorry for the confusion.